.github/workflows/CI.yml

```diff
@@ -554,12 +554,16 @@ jobs:
           rm '/usr/local/bin/2to3-3.12'
```

Comment:
@ReenigneArcher I'm not familiar with macos but isnt the
`rm '/usr/local/bin/2to3-3.13'`
missing here ?

Reply:
I did what was necessary to get CI passing. This has no effect on the final homebrew package provided to users.